ext/libdatadog_api/datadog_ruby_common.h

```diff
@@ -41,6 +41,13 @@ static inline ddog_CharSlice char_slice_from_ruby_string(VALUE string) {
   return char_slice;
 }
 
+static inline VALUE log_warning(VALUE error) {
+  VALUE datadog_module = rb_const_get(rb_cObject, rb_intern("Datadog"));
+  VALUE logger = rb_funcall(datadog_module, rb_intern("logger"), 0);
+
+  return rb_funcall(logger, rb_intern("warn"), 1, error);
+}
```

Comment:
Minor: Is it an error, or a warning? ;)

```suggestion
static inline VALUE log_warning(VALUE warning) {
  VALUE datadog_module = rb_const_get(rb_cObject, rb_intern("Datadog"));
  VALUE logger = rb_funcall(datadog_module, rb_intern("logger"), 0);

  return rb_funcall(logger, rb_intern("warn"), 1, warning);
}
```

Reply:
Repaced by warning in a9116be1e39ad0f52a0c15d56678a046f8f20206